Feature: Add GraphQL-MD plugin for Docusaurus (#157)
aragon/developer-portal: docs/subgraph/index.md

---
title: Aragon Subgraph [WIP]
sidebar_label: Introduction
sidebar_position: 0
---

## The On-Chain Information

> In Aragon we use [The Graph](https://thegraph.com/docs/en/about/), a decentralized protocol for indexing and querying blockchain data. The Graph makes it possible to query data that is difficult to retrieve directly from the Smart Contracts.

Reviewer: ```suggestion
Subgraph is a protocol for indexing and querying event data emitted from the smart contracts.
```
https://www.alchemy.com/overviews/what-is-a-subgraph

The Graph is a decentralized subgraph API. We use Alchemy subgraphs which is a more centralized version that is cheaper.
Both index events that are stored in the transaction logs on the blockchain. Please suggest more improvements @Rekard0.

The Aragon Subgraph stores all DAO-related information, either to help users to manage and interact with their DAOs or to query useful information, such as its treasury information, members, plugins, etc.

> On this site, can be found help with [queries](./01-queryExample/index.md) as well as all the documentation for our [subgraph schema](./reference-guide/index.md)

Reviewer: Something is wrong with this sentence

## Playground

> We encourage users to test the [Aragon subgraph-satsuma playground](https://subgraph.satsuma-prod.com/aragon/osx-mainnet/playground) to fully understand what information is available and how to access it.

Reviewer: ```suggestion
You can use the [Aragon subgraph-satsuma playground](https://subgraph.satsuma-prod.com/aragon/osx-mainnet/playground) for testing.
```

Sometimes speaking directly to the user (using "you") instead of using the more distanced "we" is clearer and sounds less distanced.

> Query examples to illustrate how information can be queried are provided in the [Query Examples site](./01-queryExample/index.md).

Reviewer: ```suggestion
Examples queries are provided in the [Query Example](./01-queryExample/index.md) section.
```
Should "Query Example" be renamed to "Query Examples"?
